id: 7
ns: 'quad7'
R_cam_ego: [0., -1., 0., 0.,  0., -1., 1., 0.,  0.]
t_cam_ego: [0.,0.,0.]
target_bound_box_l: 0.27
target_bound_box_w: 0.27
target_bound_box_h: 0.13
